structwrapping/structwrapping_test.go

```diff
@@ -0,0 +1,165 @@
+package structwrapping
+
+import (
+	"testing"
+
+	wrapping "github.com/hashicorp/go-kms-wrapping"
+	"github.com/hashicorp/go-kms-wrapping/wrappers/aead"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStructWrapping(t *testing.T) {
+	wrapper := aead.NewWrapper(nil)
+	wrapper.SetConfig(map[string]string{"aead_type": "aes-gcm", "key": "QmjueU/LMsZAO+rvSUMcpkBziCD5ON7BgxVqcZ6+TCI="})
+
+	t.Run("shared encryption/decryption tests", func(t *testing.T) {
+		t.Run("bad basic values", func(t *testing.T) {
```

Comment:
I always forget to use subtests like this.  ty for the reminder.

Reply:
:-)